src/helics/core/helicsTime.hpp

```diff
@@ -39,12 +39,6 @@ constexpr Time cBigTime = Time{static_cast<int64_t>(HELICS_BIG_NUMBER * 1000000)
 /** common definition of currentTime in initialization mode*/
 constexpr Time initializationTime = negEpsilon;
 
-/** user defined literal for a time variable*/
-constexpr Time operator"" _t(long double val)
-{
-    return {static_cast<double>(val)};
-}  // NOLINT
-
```

Comment:
Is this function part of the public API?

Reply:
yes,  though like the above generated several warnings, is not very useful, and was generally confusing and not used other than a test case which I changed.

Reply:
but a changelog note is warranted.